modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1266,6 +1266,30 @@ TEST_P(SqrtTest, AccuracyTest)
     }
 }
 
+TEST_P(WarpPerspectiveTest, AccuracyTest)
+{
+    cv::Point center{in_mat1.size() / 2};
+    cv::Mat rm = cv::getRotationMatrix2D(center, angle, scale);
+    cv::Matx33d transform_mat(rm.at<double>(0, 0), rm.at<double>(0, 1), rm.at<double>(0, 2),
+                              rm.at<double>(1, 0), rm.at<double>(1, 1), rm.at<double>(1, 2),
+                              0                  , 0                  , 1                 );
```

Comment:
So in fact you concatenate a `{0,0,1}` vector with a 3x2 mat to get a 3x3 mat.

I believe OpenCV provides a better way to do that :)

Reply:
Done

Reply:
Not sure that new version is simpler for review.
To reduce amount of code you may use this "overload" here:
```
Matx23d getRotationMatrix2D_(Point2f center, double angle, double scale);
```